Project2015To2017.Console/Program.cs

```diff
@@ -40,6 +41,16 @@ private static void ConvertProject(Options options)
 
 			System.Console.Out.Flush();
 
+			var analyzer = new Analyzer();
```

Comment:
Did you not make the options a constructor parameter on purpose?

Reply:
No, actually it's not important. I can change that.